cypress/integration/plugins/index-management-dashboards-plugin/indices_spec.js

```diff
@@ -256,8 +256,7 @@ describe('Indices', () => {
   });
 
   describe('can search with reindex & recovery status', () => {
-    const reindexedIndex =
-      'reindex_opensearch_dashboards_sample_data_ecommerce';
+    const reindexedIndex = 'reindex_sample_data_ecommerce';
```

Comment:
Can we randomize the name to avoid such cases in the future?

Reply:
This name is random enough I think.

Reply:
By randomize it, I mean something like `reindex_sample_data_ecommerce_${Date.now()}`. This gives you extra confidence on the uniqueness of the index name.

Reply:
> By randomize it, I mean something like `reindex_sample_data_ecommerce_${Date.now()}`. This gives you extra confidence on the uniqueness of the index name.

+1, or prefix it with `ism-` at least since re-indexing is a pretty common operation and the sample data is shared. It's probably unlikely someone will have the same name in the near future but if in a couple years it can be obfuscating something.

Reply:
Sorry, it took me some time to get to this.
Added the date.now as prefix since the suffix won't address the issue of there was some index starting with same name.